samples/exampleMainGuestList.csv

```diff
@@ -11,3 +11,4 @@ cqwas43,cc00ff88,Index,tyler32bit@gmail.com,Babyface Suite (1 King Bed)
 cq12k43,cc00ff88,Index,tyler32bit@gmail.com,Babyface Suite (1 King Bed)
 iamanid,Pez,Mike,mpesaven@gmail.com,Babyface Suite (1 King Bed)
 notanid,Pez,Mike,mpesaven@gmail.com,Babyface Suite (1 King Bed)
+bizbaz,Gadget,Hiiam,jonafree@gmail.com,Knew Management Suite (1 King Bed)
```

Comment:
Ah - makes sense we need to have these in multiple files. We might want to update the `README` with some more details on the data format(s).

Reply:
Yea we need to verify this format with the latest secret party exports